plugins/scaffolder/src/components/OngoingTask/ContextMenu.tsx

```diff
@@ -70,27 +67,13 @@ export const ContextMenu = (props: ContextMenuProps) => {
     onStartOver,
     onToggleLogs,
     onToggleButtonBar,
-    taskId,
   } = props;
   const { getPageTheme } = useTheme();
   const pageTheme = getPageTheme({ themeId: 'website' });
   const classes = useStyles({ fontColor: pageTheme.fontColor });
-  const scaffolderApi = useApi(scaffolderApiRef);
-  const analytics = useAnalytics();
   const [anchorEl, setAnchorEl] = useState<HTMLButtonElement>();
   const { t } = useTranslationRef(scaffolderTranslationRef);
 
-  const [{ status: cancelStatus }, { execute: cancel }] = useAsync(async () => {
```

Comment:
re-using the logic of the parent component instead. Nothing changes, the same event is fired